### DISCUSSION TITLE
I can't login with my username or email address.

### DISCUSSION THREAD

**Discussion by 3CH0PR3** [community/community#141094] [Other Feature Feedback, Questions, & Ideas] [answered]
Hola,

Estoy experimentando un problema grave con mi cuenta de GitHub, y no puedo acceder a ella. Hace poco realicé algunos cambios en la configuración de mi cuenta, específicamente en la sección de organizaciones. Me uní a una organización que había creado previamente con una cuenta externa. Al guardar los cambios, fui desconectado automáticamente. Ahora, cuando intento iniciar sesión, el sistema me dice que mi usuario o contraseña son incorrectos.

Intenté restablecer la contraseña, pero recibo el siguiente mensaje: "That address is either invalid, not a verified primary email or is not associated with a personal user account. Organization billing emails are only for notifications."

Este problema es muy grave para mí, ya que tengo proyectos importantes almacenados en mi cuenta. Agradecería mucho de sus ayuda para recuperar el acceso lo antes posible.

**Comment by queenofcorgis** [comment, marked as answer]
Hi @3chopre, 

This community is managed by GitHub employees, but it is not a support forum. The only place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact).

This is a public forum, and the community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. 

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion. We appreciate your understanding!

_Please note: You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them._
